include/enums.h

```diff
@@ -1402,6 +1409,7 @@ enum ActionStates {
     ACTION_STATE_IDLE                           = 0x00000000,
     ACTION_STATE_WALK                           = 0x00000001,
     ACTION_STATE_RUN                            = 0x00000002,
+    // all states above are considered locomotion states
```

Comment:
By which func(s)? Would be good to reference it

Reply:
ill add them as i find them. its pretty common in the player action states / physics functions iirc.